.github/workflows/build_wamr_vscode_ext.yml

```diff
@@ -32,11 +32,14 @@ jobs:
         working-directory: test-tools/wamr-ide/VSCode-Extension
 
       - name: generate wamr ide vscode extension
+        env: 
+          credentials: ${{ secrets.TOKEN }}
         run: |
           npm install -g vsce
           rm -rf node_modules
           npm install
           vsce package
+          vsce publish -p ${{ secrets.TOKEN }}
```

Comment:
does it mean the new version will be published on every build? If so, in general I like the idea of CI/CD, but I wonder if we need to put some extra checks in the extension to make sure customers use the correct version of the WAMR <-> vscode extension; I guess there might be features we add to unreleased version of WAMR that will break the use of the extension or make it non functioning correctly.